.github/workflows/load-tests.yml

```diff
@@ -96,6 +96,7 @@ jobs:
         env:
           TEST_ARGS: "-test.run=${{ matrix.test }}"
       - name: Set results filename
+        if: ${{ failure() || success() }}
```

Comment:
I don't know much about the fine details of this vs `always()`. Do you think we could use `always()` here? (https://docs.github.com/en/actions/learn-github-actions/expressions#example-of-always)

Reply:
i thought about that as well and opted for using the same expression that is used in other places.
The main difference is that `always()` also runs when a job is cancelled which is probably not needed here.

Reply:
Makes sense to be consistent, thanks for clarifying